admin-core/assets/src/common/block-icons.js

```diff
@@ -727,6 +727,33 @@ const UAGB_Block_Icons = {
 			strokeWidth: 1.4,
 		} ),
 	),
+	'gbs': el(
```

Comment:
Preferrably rename this so that any other developer in the future is aware what this icon is for.

```suggestion
	'global-block-styles': el(
```

Reply:
Renamed